editoast/src/converters/osm_to_railjson.rs

```diff
@@ -110,12 +110,12 @@ pub fn parse_osm(osm_pbf_in: PathBuf) -> Result<RailJson, Box<dyn Error + Send +
             (4, 4) => railjson
                 .switches
                 .push(double_slip_switch(node, &adj.branches)),
-            _ => log::debug!("node {id} with {edges_count} edges and {branches_count} branches"),
```

Comment:
Does the output look any different? Do we know if we currently consume it?

Reply:
So, the output should not look different... but I realised it doesn't work (I've made that a few times before on other projects so I didn't check). Theoritically, it should just work because `trancing-subscriber` has a default feature `tracing-log` that actually catch `log` events and transform them into `tracing` events, and therefore, would end up on the console. But I can't make that work even on a minimal example. I found [an old issue](https://github.com/tokio-rs/tracing/issues/489) about that but it should be resolved. I'm still investigating.

Reply:
That said, maybe I don't need to lose time in this investigation if we merge https://github.com/osrd-project/osrd/pull/6650 first 🤷

Reply:
OK, just me forgetting the basics 🤦 I forgot the `.init()` so no logger was listening to the `log` records.

Reply:
I also changed from `.pretty()` to `.compact()` to keep one-line logs. See description of the PR which I edited to put some screenshots.

Reply:
Tested. Before, it was like this (you'll have to trust me I ran that one on `dev`)

![image](https://github.com/osrd-project/osrd/assets/2520723/22f05cc0-e7b3-4988-ae46-43cb399f80d7)

And after (ran on my branch)

![image](https://github.com/osrd-project/osrd/assets/2520723/a1cd7ff3-1b21-4135-b6bc-e216a01dd848)